systemtests/tests/strippath/testrunner-01-setup

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
You should split up this file into a file called `test-setup` (where all the setup is) and a file called something like `testrunner-01-backup` (where only the backup happens).

Reply:
Done